tests/lightning/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
The tests should ideally under `tests/trainers` directory to maintain the current test directory structure. You can keep the `lightning' subdirectory.